src/app/pages/BuyPage/index.module.scss

```diff
@@ -0,0 +1,16 @@
+// new swap page
+.swap-page {
+  max-width: 100%;
```

Comment:
update classes to use `tailwind css` as much as possible.

Reply:
fixed